tests/test_regrid.py

```diff
@@ -27,20 +27,22 @@
 @pytest.fixture(scope="session")
 def sample_input_data() -> xr.Dataset:
     ds = xr.open_dataset(DATA_PATH / "era5_2m_dewpoint_temperature_2000_monthly.nc")
-    return ds.isel(time=slice(0, N_TIMESTAMPS)).persist()
+    # slice after compute due to current xarray bug: https://github.com/pydata/xarray/issues/8909
+    # then convert to dask so regridding is lazy on attribute tests
+    return ds.compute().isel(time=slice(0, N_TIMESTAMPS)).chunk()
```

Comment:
From reading that issue it seems like we can also avoid the bug by installing netCDF4 as dependency? We can add it to the dev dependencies.